pom.xml

```diff
@@ -1733,7 +1733,7 @@
               <configuration>
                 <rules>
                   <requireMavenVersion>
-                    <version>${maven.version}</version>
+                    <version>[3.3,)</version>
```

Comment:
Actually, that's what the existing specification already means:
https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html